testSrc/unit/io/flutter/utils/UrlUtilsTest.java

```diff
@@ -0,0 +1,20 @@
+package io.flutter.utils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class UrlUtilsTest {
+    @Test
+    public void testAddUrlTags() {
+        assertEquals(
```

Comment:
nit: add test with multiple links.

Reply:
There's one example of this on line 16; let me know if you're thinking of something else though.